test/unit/service_test.rb

```diff
@@ -650,6 +650,16 @@ def test_gateways
       refute_includes gateways, 'service_mesh_istio'
     end
 
+    def test_gateways_with_apicast_custom_url
+      ThreeScale.config.stubs(apicast_custom_url: true)
```

Comment:
This won't really change the test, right?

Reply:
I understand the idea is to show that even with `apicast_custom_url: true` it now shows the `self_managed` option. Maybe also showing that the test passes with `apicast_custom_url: false`?

Reply:
It is tested above already